crates/iceberg-catalog/src/service/mod.rs

```diff
@@ -84,6 +84,12 @@ pub struct TableId(uuid::Uuid);
 #[serde(transparent)]
 pub struct WarehouseId(pub(crate) uuid::Uuid);
 
+impl Default for WarehouseId {
+    fn default() -> Self {
+        Self(uuid::Uuid::now_v7())
+    }
+}
+
```

Comment:
I'd prefer not using `default()` as a constructor which returns a random value. Let's maybe use something like `new_random()`. Using `Default` can make it seem like there's a default warehouse id